src/libc/time.rs

```diff
@@ -271,10 +271,8 @@ unsafe impl SafeRead for timeval {}
 fn gettimeofday(
     env: &mut Environment,
     timeval_ptr: MutPtr<timeval>,
-    timezone_ptr: MutVoidPtr, // deprecated, always NULL
+    _timezone_ptr: MutVoidPtr, // deprecated, not used
 ) -> i32 {
-    assert!(timezone_ptr.is_null());
```

Comment:
This is too hacky to me :p

Can you instead check what the actual timezone struct format is for iPhone OS (tbh I am lazy and just look at the macOS headers…), and return one that's all zeroes?

Reply:
Looks like it is:

```c
struct timezone {
    int     tz_minuteswest; /* minutes west of Greenwich */
    int     tz_dsttime;     /* type of dst correction */
};
```

so it can be:

```rust
#[allow(non_camel_case_types)]
#[repr(C, packed)]
struct timezone {
    tz_minuteswest: i32,
    tz_dsttime: i32,
}
```

and then just set to zero

Reply:
Done